rosdep/python.yaml

```diff
@@ -4915,6 +4915,24 @@ python-trimesh-pip:
   ubuntu:
     pip:
       packages: [trimesh]
+python-twilio:
+  debian:
+    buster: [python-twilio]
+    jessie:
+      pip:
+        packages: [twilio]
+    stretch:
+      pip:
+        packages: [twilio]
```

Comment:
Let's not bother with jessie or stretch here, as they are both unsupported: https://wiki.debian.org/LTS